server/ethereum/ethereum.test.js

```diff
@@ -213,7 +213,8 @@ test('handleListenEvent throws error with empty params', async done => {
   done()
 })
 
-test('handlePinHashEvent pins file of cid it was passed', async done => {
+test(`handlePinHashEvent pins file of cid it was
+      passed and removes it from database`, async done => {
```

Comment:
Small note, but it doesn't actually pin any cid. Just removes it from DB. Prob more accurate to say "handlePinHashEvent removes file from database to avoid garbage colleection" or something along those lines